src/prepare_dlc_dev_environment.py

```diff
@@ -66,17 +56,24 @@ def get_args():
         default=VALID_TEST_TYPES,
         help="Types of tests to run",
     )
-    parser.add_argument(
+    buildspec_parser = parser.add_argument_group("buildspec options")
```

Comment:
if introducing a new group, you can update the currency option to use buildspec_parser as well